src/pages/Lesson.vue

```diff
@@ -25,24 +25,23 @@
         :createTestFile="logic.options.createTestFile"
         :createTestTree="logic.options.createTestTree"
         :lessonId="parseInt(lessonId, 10)"
-        :lessonTitle="lesson.title"
+        :lessonTitle="!!lesson && lesson.meta.title"
     />
     <MultipleChoiceLesson
-        v-else-if="lesson.type === 'multiple-choice'"
-        :text="text"
+        v-else-if="!!lesson && lesson.meta.type === 'multiple-choice'"
+        :text="!!esson && lesson.html"
```

Comment:
I dont know what this does but you probably dont want `esson` here. Can you add a cypress test to show the changes you expect to see from these?

Reply:
The component would crash always on the `lesson.meta.title` first, so I only added this in case vue internals change.

```suggestion
        :text="!!lesson && lesson.html"
```

Reply:
Would it be possible to write a cypress test for the logic in this file? it's not immediately obvious how it all fits together and it wasn't clear that one of the conditions was broken. Would be great to have a test that fails rather than relying on human eyes to catch typos.

Reply:
Ah yes, will do 👍

Reply:
Pushed a test for this here: https://github.com/ProtoSchool/protoschool.github.io/pull/521/commits/eaa1c236ad58795be7529931047344e6154b13a2

Good catch on this. Lesson's with multiple choice were not rendering the lesson content and we weren't testing for that.

At first I thought I could simply do snapshot testing, but then every time a typo or any other content parts change in the future, the tests would fail.
Another approach I tried was to assert the rendered html, but because of the sanitized result of the html of vue's `v-html`, it was hard to reproduce the same resulting html.

For this test assertion I ended up adding the resulting html from the markdown (result of `marked()`) to a `data-cy-text` attribute so that we could assert the text ended up correctly in the element (`<div class="lesson-text" v-html="text" :data-cy-text="text">`). This way we can check the loaded markdown string is the same as the one rendered in `v-html`.

Let me know if you know of a better way to test this.